scripts/summarize_celestia_sync_runs.py

```
@@ -9,7 +9,7 @@
 import re
 import sys
 from pathlib import Path
-from typing import Any
+from typing import Any, Callable, Iterable
```

Ruff (0.15.20)

[warning] Import from `collections.abc` instead: `Callable`, `Iterable`

Import from `collections.abc`

(UP035)